### PR TITLE
feat: zookeeper support  mutil nodes

### LINKED ISSUE #53: zookeeper should support multiple nodes
https://github.com/api7/apisix-seed/blob/main/internal/discoverer/zookeeper.go#L110-L116

### PR DESCRIPTION
**Pull request by soulbird**
Fixes: #53

### CODE REVIEW

File: test/e2e/tools/regcenter/zookeeper.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ func NewZookeeper() *Zookeeper {
 }
 
 func (zookeeper *Zookeeper) Online(node *common.Node) error {
-	nodeStr := `{"host":"` + common.DOCKER_GATEWAY + `","port":` + node.Port + `}`
+	nodeStr := `[{"host":"` + common.DOCKER_GATEWAY + `","port":` + node.Port + `}]`
```
Comment (spacewander) [inline test/e2e/tools/regcenter/zookeeper.go:28]: Can we add a test case with multiple nodes?

File: internal/discoverer/zookeeper_test.go [hunk 2]
```diff
@@ -96,14 +96,14 @@ func TestZookeeperDiscoverer(t *testing.T) {
 	msgChan := dis.Watch()
 
 	// create service
-	err = createZkService(conn, svcPath, `{"host":"127.0.0.1","port":1980,"weight":100}`)
+	err = createZkService(conn, svcPath, `[{"host":"127.0.0.1","port":1980,"weight":100}]`)
```
Comment (spacewander) [inline internal/discoverer/zookeeper_test.go:99]: Is it possible to provide multi nodes in the test data, like `[{"host":"127.0.0.1","port":1980,"weight":100}], ...`?

Comment (soulbird) [reply internal/discoverer/zookeeper_test.go:99]: Complex arrays are difficult to compare when using json, and using `JSONEq` directly will make the test unstable.

Comment (spacewander) [reply internal/discoverer/zookeeper_test.go:99]: What about sorting the output before the comparison?

Comment (soulbird) [reply internal/discoverer/zookeeper_test.go:99]: The serialization of JSON is deeply hidden, and it is difficult to intervene. And that part of the code is not used for testing, the sorting in it is not appropriate. Look: https://github.com/api7/apisix-seed/blob/main/internal/discoverer/zookeeper_test.go#L56

**Review by spacewander — APPROVED** [review]

**Pull request merged**